models/public/quartznet/quartznet.md

```diff
@@ -0,0 +1,75 @@
+# quartznet
+
+## Use Case and High-Level Description
+
+QuartzNet model is a pair of encoder and decoder. QuartzNet’s design is based on the Jasper architecture,
+which is a convolutional model trained with Connectionist Temporal Classification (CTC) loss.
+The model was trained in NeMo on multiple datasets: LibriSpeech,
+Mozilla Common Voice, WSJ, Fisher, Switchboard, and NSC Singapore English.
+For details see [repository](https://github.com/NVIDIA/NeMo), [paper](https://arxiv.org/pdf/1910.10261.pdf).
+
+## Accuracy
+
+| Metric                       | Value |
+| ---------------------------- | ----- |
+| WER @ Librispeech test-clean | 3.86% |
+
+## Specification
+
+| Metric           | Value              |
+| ---------------- | ------------------ |
+| Type             | Speech recognition |
+| GFLOPs           | 2.4195             |
+| MParams          | 18.8857            |
+| Source framework | PyTorch\*          |
+
+### Input
+
+#### Original model
+
+Audio signal, name - `audio_signal`,  shape - `1, 64, 128`, format is `B, N, C` where:
+
+- `B` - batch size
+- `N` - number of audio frames
+- `C` - clip duration
+
+#### Converted model
+
+The converted model has the same parameters as the original model.
+
+### Output
+
+#### Original model
+
+Per-frame probabilities (after LogSoftmax) for every symbol in the alphabet, name - `output`,  shape - `1, 64, 29`, output data format is `B, N, C` where:
+
+- B - batch size
+- N - number of audio frames
+- C - alphabet size, including the CTC blank symbol
+
+The per-frame probabilities are to be decoded with a CTC decoder.
+The alphabet is: 0 = space, 1...26 = "a" to "z", 27 = apostrophe, 28 = CTC blank symbol.
```

Comment:
maybe we can provide the file with alphabet?

Reply:
We have one in [speech_recognition_demo/python/default_alphabet_example.conf](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/demos/speech_recognition_demo/python/default_alphabet_example.conf). Added link.